client/checkout/woopay/email-input-iframe.js

```diff
@@ -305,32 +298,29 @@ export const handlePlatformCheckoutEmailInput = async (
 	}
 
 	const dispatchUserExistEvent = ( userExist ) => {
-		const PlatformCheckoutUserCheckEvent = new CustomEvent(
-			'PlatformCheckoutUserCheck',
-			{
-				detail: {
-					isRegisteredUser: userExist,
-				},
-			}
-		);
-		window.dispatchEvent( PlatformCheckoutUserCheckEvent );
+		const WooPayUserCheckEvent = new CustomEvent( 'WooPayUserCheck', {
```

Comment:
I know this approach has been taken from the previous implementation, but I don't see a reason for the event name to start with the capital letter. 
```suggestion
		const woopayUserCheckEvent = new CustomEvent( 'woopayUserCheck', {
```